warrior/Actions/CliActions/cli_actions.py

```diff
@@ -335,49 +340,53 @@ def connect_ssh(self, system_name, session_name=None, prompt=".*(%|#|\$)",
                 # Create an object for WarriorCli class and use it to
                 # establish ssh sessions
                 wc_obj = WarriorCli()
-                if credentials['conn_type'] == "SSH_NESTED":
```

Comment:
The change here is to move the warrior_cli_class logic inside a block that only get executed if it is not in mock/sim mode